Modules/Sources/Storage/GRDB/Model/PersistedProductVariationAttribute.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import GRDB
+
+public struct PersistedProductVariationAttribute: Codable {
+    public private(set) var id: Int64?
```

Comment:
Similar question about manually updating the id. Feel free to just respond to one.